Resources/doc/cookbook/populate-optimizations.md

```diff
@@ -0,0 +1,16 @@
+Populate optimizations
+======================
+
+Here in this chapter we describe some ways to improve performance of `fos:elastica:populate` command. 
+
+Enqueue Elastica Bundle
+-----------------------
+
+The solution utilize [message queue](https://en.wikipedia.org/wiki/Message_queue) by distributing the work among several consumers.
+You can use different transports such as RabbitMQ, Amqp, Stomp, Filesystem and so.
+The performance gain depends on how much consumers you run. 
+For example 10 consumers may give you 5 to 7 times better performance.
```

Comment:
I agree that such promises cannot be made because it almost always depends on someone's data/machines/etc but perhaps a link to a sample benchmark would be a nice workaround. So that devs can compare that scenario to theirs.